include/fmt/base.h

```diff
@@ -2007,6 +2014,11 @@ using is_formattable = bool_constant<!std::is_base_of<
     detail::unformattable, decltype(detail::arg_mapper<buffered_context<Char>>()
                                         .map(std::declval<T&>()))>::value>;
 
+#if FMT_USE_CONCEPTS
+template <typename T, typename Char = char>
+concept formattable = is_formattable<remove_reference_t<T>, Char>::value;
```

Comment:
if we plan to have `formattable` as part of the public interface, is it possible to make it work with, for instance, the types supported by the range-alike formatter provided by {fmt}, so that following statement holds:
```c++
static_assert(fmt::formattable<std::vector<int>>);
```

please note, `is_formattable<std::vector<int>>::value` is `false`.

Reply:
If you don't use modules, you need to explicitly include `format/ranges.h` for range formatting: https://godbolt.org/z/TT55szeTa

Reply:
@matt77hias i think i failed to explain my intention clearly. i was not asking on how to format ranges. by "it" i mean the concept of `formattable`, not `fmt::formatter`.

lemme rephrase, is it possible to make `formattable` to work with range types , so that, for instance, `std::vector<int>` can fulfill the constraint of `formattable`?